packages/instant/src/components/ui/container.tsx

```diff
@@ -64,6 +67,12 @@ export const Container =
     ${props => (props.hasBoxShadow ? `box-shadow: 0px 2px 10px rgba(0, 0, 0, 0.1)` : '')};
     background-color: ${props => (props.backgroundColor ? props.theme[props.backgroundColor] : 'none')};
     border-color: ${props => (props.borderColor ? props.theme[props.borderColor] : 'none')};
+    &:hover {
+        ${props =>
+            props.darkenOnHover
+                ? `background-color: ${darken(0.05, props.theme[props.backgroundColor || 'white'])}`
```

Comment:
the default background-color is `none`, but here we treat the default as `white` 🤔